pkgs/development/libraries/ffmpeg/generic.nix

```
@@ -441,6 +439,7 @@ stdenv.mkDerivation (finalAttrs: {
      *  External libraries
      */
     (enableFeature withAlsa "alsa")
+    (optionalString withAribcaption "--enable-libaribcaption")
```

Shouldn't this be `enableFeature`?
```suggestion
    (enableFeature withAribcaption "--enable-libaribcaption")
```